xml/installation-ardana-manila.xml

```diff
@@ -0,0 +1,266 @@
+<?xml version="1.0"?>
+<!DOCTYPE chapter [
+ <!ENTITY % entities SYSTEM "entities.ent"> %entities;
+]>
+<chapter xml:id="install_ardana_manila"
+ xmlns="http://docbook.org/ns/docbook" version="5.1"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink">
+ <title>Installing &o_sharefs; and Creating &o_sharefs; Shares</title>
+ <section>
+  <title>Installing &o_sharefs;</title>
+  <para>
+   The &ostack; Shared File Systems service (&o_sharefs;) provides file storage
+   to a virtual machine. The Shared File Systems service provides a storage
+   provisioning control plane for shared or distributed file systems. The
+   service enables management of share types and share snapshots if you have a
+   driver supports them.
+  </para>
+  <para>
+   The &o_sharefs; service consists of the following components: manila-api,
+   manila-data, manila-scheduler, manila-share, and messaging queue.
+  </para>
+  <para>
+   The following steps will install &o_sharefs;:
+  </para>
+  <procedure>
+   <step>
+    <para>
+     Log in to the &lcm;.
+    </para>
+   </step>
+   <step>
+    <para>
+     Apply &o_sharefs; changes in <filename>control_plane.yml</filename>.
+    </para>
+    <screen>&prompt.ardana;cd /var/lib/ardana/openstack/my_cloud/definition/data/</screen>
+    <para>
+     Add <literal>manila-client</literal> to the list of service components for
+     &lcm;, <literal>manila-api</literal> to the &contrnode;, and
+     <literal>manila-share</literal> to &slsa; &compnode;
+    </para>
+   </step>
+   <step>
+    <para>
+     Run the Configuration Processor.
+    </para>
+    <screen>&prompt.ardana;cd ~/openstack
+&prompt.ardana;git add -A
+&prompt.ardana;git commit -m "Manila config"
+&prompt.ardana;cd ~/openstack/ardana/ansible/
+&prompt.ardana;ansible-playbook -i hosts/localhost config-processor-run.yml
+&prompt.ardana;ansible-playbook -i hosts/localhost ready-deployment.yml</screen>
+   </step>
+   <step>
+    <para>
+     Deploy &o_sharefs;
+    </para>
+    <screen>&prompt.ardana;cd ~/scratch/ansible/next/ardana/ansible
+&prompt.ardana;include manila-deploy.yml in ardana-deploy.yml
```

Comment:
In the [Deploy] section, instead of:
```
include manila-deploy.yml in ardana-deploy.yml
ansible-playbook -i hosts/verb_hosts site.yml
```
Change the two steps to:
```
ansible-playbook -i hosts/verb_hosts manila-deploy.yml
ansible-playbook -i hosts/verb_hosts ardana-reconfigure.yml
```